examples/src/indexes/vector_stores/vectara_search.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { VectaraStore, VectaraLibArgs } from "langchain/vectorstores/vectara";
+import { FakeEmbeddings } from "../../../../langchain/embeddings.js";
+
+export async function run() {
+    // Create a store and fill it with some texts + metadata
+    const args: VectaraLibArgs = {
+        customer_id: Number(process.env.VECTARA_CUSTOMER_ID) ?? 0,
+        corpus_id: Number(process.env.VECTARA_CORPUS_ID) ?? 0,
+        api_key: process.env.VECTARA_API_KEY ?? ""
+    };
+    const store = await VectaraStore.fromTexts(
+        ["hello world", "hi there", "how are you", "bye now"],
+        [{ foo: "bar" }, { foo: "baz" }, { foo: "qux" }, { foo: "bar" }],
+        new FakeEmbeddings(),       // Vectara has its own embeddings, so we don't need to get them separately
```

Comment:
Can we remove this parameter from the interface entirely, and consume `FakeEmbeddings` directly from within `vectara.ts`?

I'm not sure if that will feel strange to people who are used to consuming other vectorstores, but from a naive perspective it seems like simplifying the interface this way will make it easier to use.

Reply:
We cannot remove this from interface. This method is inherited from base class.